### ISSUE TITLE
resume module

### ISSUE THREAD

**Issue by pamela0530** [KevinMusgrave/pytorch-metric-learning#340]
could you add a resume module to the project ?

**Comment by KevinMusgrave** [comment]
What do you mean by "resume module"?

**Comment by pamela0530** [comment]
> What do you mean by "resume module"?

 oh, I have found an API called “load_latest_saved_models”.  

A little problem is that in subsequent training, logs start again at Epoch 1

**Comment by KevinMusgrave** [comment]
I think this will make the logs start at the correct epoch:

```python
start_epoch = hooks.load_latest_saved_models(trainer, model_folder, device)
trainer.train(start_epoch, num_epochs)
```
